sphinx_needs/lsp/esbonio.py

```diff
@@ -242,7 +242,8 @@ def get_lines(ls: NeedlsFeatures, params: Union[CompletionContext, DefinitionCon
 
 def get_word(ls: NeedlsFeatures, params: Union[CompletionContext, DefinitionContext, HoverContext]) -> str:
     """Return the word in a line of text at a character position."""
-    line_no, col = params.position
+    line_no = params.position.line
```

Comment:
I guess this is not backward compatible, or?

Thinking about something like:
```python

if pyglsversion < 123:
    line_no, col = params.position
else:
    line_no = params.position.line
    col = params.position.character
```

Reply:
In the old implementation `params.position` is just a shortcut for `params.position.__iter__` which returns  `(params.position.line, params.position.character)`.
See [here](https://github.com/openlawlibrary/pygls/blob/fb80b8c40bc1e92a15f03ab79316abd5712fbd4b/pygls/lsp/types/basic_structures.py#L174-L175).
But I can make it more explicit, if that's better for you.

Reply:
Yeah, lets make it more explicit.

I also thinks the CI stuff got fixed, and the PR needs to be rebased.
Then everything should be good.

Reply:
Can we merge this one already?
Would be nice to have a 1.3.0 release with this fix.

Reply:
Sorry, I got a little swamped lately. I'll update tomorrow.